GP350.py

```diff
@@ -0,0 +1,54 @@
+from pymeasure.instruments import Instrument, Channel
+import numpy as np
+
+
+class GP_PressureChannel(Channel):
+    """
+    Channel of the GP350UHV vacuum sensor for pressure measurement.
+
+    Each channel corresponds to a filament or display output of the sensor.
+    """
+
+    pressure = Instrument.measurement(
+        "#RD{ch}",
+        """Measure the current pressure in mbar (float).""",
+        preprocess_reply=lambda reply: reply.lstrip("* ").strip(),
+        get_process=lambda r: np.nan if float(r) == 9.90e9 else float(r),
+    )
+
+
+class GP350UHV(Instrument):
+    """
+    Representation of the GP350UHV Vacuum Sensor.
+
+    Provides access to multiple pressure channels corresponding to filaments
+    and display outputs. Each channel can be queried for its current pressure
+    in mbar.
+    """
+
+    def __init__(self, adapter, name="GP350UHV Vacuum Sensor", **kwargs):
+        """
+        Initialize the GP350UHV instrument.
+
+        Parameters
+        ----------
+        adapter : Adapter
+            Communication adapter (e.g., SerialAdapter, VISAAdapter).
+        name : str, optional
+            Name of the instrument, default is "GP350UHV Vacuum Sensor".
+        **kwargs
+            Additional keyword arguments passed to the Instrument base class.
+        """
+        super().__init__(adapter, name, includeSCPI=False, **kwargs)
+
+    filament_1 = Instrument.ChannelCreator(GP_PressureChannel, "1")
+    """Measure the pressure from filament 1 in mbar (float)."""
```

Comment:
It is a channel and not a measurement, so the comment is wrong.

I would just state, in a single comment above the channels that 1,2 are filaments and A,B the display lines.